testdata/zk/Constructor.t.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.18;
+
+import "ds-test/test.sol";
+import "../cheats/Vm.sol";
+import "../default/logs/console.sol";
+
+contract Bank {
```

Comment:
I'd reuse the `Bank` contract, either from it's on file here and then importing it in the script project or the opposite